package/network/services/igmpproxy/Makefile

```diff
@@ -39,13 +39,7 @@ define Package/igmpproxy/conffiles
 /etc/config/igmpproxy
 endef
 
-TARGET_CFLAGS += -Dlog=igmpproxy_log
-
-define Build/Compile
-	$(MAKE) -C $(PKG_BUILD_DIR)/src \
-		CC="$(TARGET_CC)" \
-		CFLAGS="$(TARGET_CFLAGS) -std=gnu99"
-endef
+TARGET_CFLAGS += -Dlog=igmpproxy_log -std=gnu99
```

Comment:
`-std=gnu99` should not be required as `./configure` automatically adds it if needed:
https://github.com/pali/igmpproxy/blob/0.3/configure.ac#L6

And I'm not sure what is `-Dlog=igmpproxy_log` doing... igmpproxy does not have any `log` function.

Reply:
Thx. It looks like leftovers related to the previous version. I removed it and it compiles and works ok on Omnia master.